menas/src/main/scala/za/co/absa/enceladus/menas/controllers/RestExceptionHandler.scala

```diff
@@ -56,6 +56,11 @@ class RestExceptionHandler {
     ResponseEntity.notFound().build[Any]()
   }
 
+  @ExceptionHandler(value = Array(classOf[EndpointDisabled]))
+  def handleEndpointDisabled(exception: EndpointDisabled): ResponseEntity[Any] = {
+    ResponseEntity.status(HttpStatus.I_AM_A_TEAPOT).build[Any]() // Could change for LOCKED but I like this more
```

Comment:
I am quite surprised no one comment on this.

Reply:
The comment in code is there, that explains it 😉